src/OnboardingSPA/data/routes/default-flow.js

```diff
@@ -615,7 +627,8 @@ export const initialTopSteps = () => {
 	const topSteps = filter( steps, ( step ) => {
 		return (
 			! step.path.includes( '/step/get-started' ) &&
-			! step.path.includes( '/step/design' )
+			! step.path.includes( '/step/design' ) &&
+			! step.path.includes( '/step/complete' )
```

Comment:
Won't it be better to have a boolean or some sort while describing a page up above rather than again and again adding such hardcoded texts? Just thinking out loud.
`isTopPage: True or False`

Reply:
I like the thinking but the problem is not so simple as we have "fake-steps" being injected in cases like `design` and `get-started` which `complete` does not need. This also means that `design` and `get-started` are "fake" top steps which simply do not have a component to render so `isTopPage` would not exactly make sense. I would say it is a good idea but something to do once we see some general pattern in this filter.